CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ set(PRODUCT_BUILDDATE "1970-01-01" CACHE STRING "Product build date")
 set(PRODUCT_SOURCE_REVISION "0000000000000000000000000000000000000000" CACHE STRING "Product source SHA")
 set(PRODUCT_SOURCE_REPOSITORY "<unspecified>" CACHE STRING "Product source branch")
 set(PRODUCT_BITS "${BUILD_BITS}bit")
+set(GEODE_AUTHEXPIREDEXCEPTION_VERSION "1.15.0-build.546" CACHE STRING "First build containing AuthenticationExpiredException")
```

Comment:
Why not just test for the existence of the exception class or not and set a "has feature" type flag?

Reply:
Just because that's a more complex fix, and neither of us really have a clue how to implement it using cmake.  Like how would we do this, go dump the class list from a jar file after we find Geode?  Which of the umpteen Geode jar files should we look in?  How do we run the jar tool at this stage via cmake?  After we do or do not find the exception class, how do we put that back into a cmake variable we can use to conditionally compile our Java?

Reply:
Ok, fair enough, but why the specific build number. When building locally on your machine, since Apache doesn't do binary releases, that build number is almost always 0. Just detect "1.15.0". We don't need to handle building sources between GAs.

Reply:
Also the GA release ends up with build 0 too.

Reply:
Good suggestion. Makes sense to use 1.15.0.